src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -109,19 +110,33 @@ export const QueryResultPane = () => {
             return;
         }
         const observer = new ResizeObserver(() => {
-            if (!gridRef.current) {
+            if (!gridRefs.current) {
                 return;
             }
             if (!ribbonRef.current) {
                 return;
             }
+
             if (gridParent.clientWidth && gridParent.clientHeight) {
-                gridRef.current.resizeGrid(
-                    gridParent.clientWidth,
-                    gridParent.clientHeight -
-                        ribbonRef.current.clientHeight -
-                        5,
-                );
+                //TODO: add case for multiple result grid
```

Comment:
Can you add a GH issue link to this comment so the TODO doesn't get lost?

Reply:
actually this is fixed, I'll remove it.